bin/acli.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * @file
+ * Acquia CLI command line front file.
+ */
+
+namespace Acquia\Cli;
+
+use Stecman\Component\Symfony\Console\BashCompletion\CompletionCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Input\ArgvInput;
+use Symfony\Component\Debug\Debug;
+use Symfony\Component\Filesystem\Filesystem;
+
+if (!file_exists(__DIR__ . '/../vendor/autoload.php')) {
+  die("Could not find autoloader. Run 'composer install' first.\n");
+}
+require __DIR__ . '/../vendor/autoload.php';
+
+ini_set('memory_limit', -1);
```

Comment:
Eek!

Reply:
ORCA does this, I assumed there's a good reason, but we don't have too.